From ada5f7957cd7b690b6637bea88720bc91aba633c Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Tue, 26 Jun 2018 16:29:31 +0300
Subject: [PATCH 0128/1239] mvebu: config: use different env addresses

The ATF RT services region and TEE overlaps with existing env addresses.
Because the ATF RT service and TEE region is going to be marked as
secure and can't be overwritten, use different different addresses for
some envs.

Change-Id: Iffdc8eb37539dde5889a3c5d4ae7ab67e0d32469
Signed-off-by: Igal Liberman <igall@marvell.com>
---
 include/configs/mvebu_armada-common.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 5b08a4f92d..2df2a5dff3 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -26,11 +26,11 @@
 #define CONFIG_HAS_ETH2
 #define CONFIG_ETHPRIME			"eth0"
 #define CONFIG_ROOTPATH			"/srv/nfs/" /* Default Dir for NFS */
-#define CONFIG_EXTRA_ENV_SETTINGS	"kernel_addr_r=0x5000000\0"	\
+#define CONFIG_EXTRA_ENV_SETTINGS	"kernel_addr_r=0x7000000\0"	\
 					"initrd_addr=0xa00000\0"	\
 					"initrd_size=0x2000000\0"	\
-					"fdt_addr_r=0x4f00000\0"	\
-					"loadaddr=0x5000000\0"		\
+					"fdt_addr_r=0x6f00000\0"	\
+					"loadaddr=0x6000000\0"		\
 					"fdt_high=0xffffffffffffffff\0"	\
 					"hostname=marvell\0"		\
 					"ramdisk_addr_r=0x8000000\0"	\
-- 
2.29.0

